requirements.txt

```diff
@@ -1,21 +1,10 @@
-# =============================================================================
-# DEPRECATION WARNING:
```

Comment:
Is this disclaimer not relevant anymore? We don't care about the fact that PyScaffold will not automatically create it anymore, but in case the requirements.txt is still only there to be parsed by github it might be useful to add a comment on that.

Reply:
Good point. done.